src/ert/workflow_runner.py

```diff
@@ -107,14 +107,16 @@ class WorkflowRunner:
     def __init__(
         self,
         workflow: Workflow,
+        config_file: str | None = None,
```

Comment:
Also commented in semeio, could potentially give reports dir, we do something similar for the update report, that has a folder it writes to.